### ISSUE TITLE
error while deploying yolov5 model using tensoflow serving. 

### ISSUE THREAD

**Issue by MrYelameli** [ultralytics/yolov5#3166]
Hi, 
I have converted the yolov5 pytorch model to tensoflow. I want to deploy it using tensorflow serving. Here is the `predict.py` file. 
Here is the script
https://gist.github.com/MrYelameli/e586941fdb0be8f9aedf7f912a60ef8f
I am getting the following error
`{'error': 'Index out of range using input dim 3; input has only 3 dims\n'
          '\t [[{{node onnx_tf_prefix_Slice_14}}]]'}`

Can anyone help with this? 
Thank you.

**Comment by glenn-jocher** [comment]
@MrYelameli see https://github.com/ultralytics/yolov5/pull/1127 for the best TF export!
